src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,57 @@
+
+.calendar {
+  $calendar-gap: 1px;
+  $calendar-padding: 10px;
+  $calendar-width: $days-in-line * $day-suqare-side +
+    $calendar-padding * 2 + $calendar-gap * ($days-in-line - 1);
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $calendar-gap;
+  padding: $calendar-padding;
+  
+  @for $i from 1 through $max-month-length {
+    & :nth-child(#{$i})::before {
+      position: absolute;
```

Comment:
You could move the loop @for to the calendar__day element and that way no need for using position: absolute;